lib/routes/routeBackbeat.js

```diff
@@ -287,18 +429,9 @@ function routeBackbeat(clientIP, request, response, log) {
                 });
                 return next(errors.MethodNotAllowed);
             }
-            if (usingMultipleBackend) {
-                if (!_isValidLocation(request)) {
-                    log.debug('invalid location constraint in request', {
-                        method: request.method,
-                        bucketName: request.bucketName,
-                        objectKey: request.objectKey,
-                        resourceType: request.resourceType,
-                    });
-                    return next(errors.InvalidRequest);
-                }
-                return backbeatRoutes[request.method][request.resourceType](
-                    request, response, bucketInfo, objMd, log, next);
+            if (request.resourceType === 'multiplebackenddata') {
+                return backbeatRoutes[request.method][request.resourceType]
```

Comment:
Did you check the validity of `request.query.operation` earlier? If not it would be good to gracefully return an error if the operation does not exist.

Reply:
Good point. Included checks for this.